catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -38,7 +38,7 @@
 
 public final class Constants {
 
-	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = 0.999991f;
+	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = 0.999992f;
```

Comment:
@wslany Should we already increase language version here ?

Reply:
If the xml structure is changed, then yes.